### PR TITLE
support for running not compiled scirpt

### PR DESCRIPTION
**Pull request by mlipok**

### CODE REVIEW

File: WhyNotWin11.au3 [hunk 1]
```diff
@@ -109,8 +109,10 @@ Func ProcessCMDLine()
 						EndSwitch
 				EndSelect
 			Case Else
-				MsgBox(0, "Invalid", 'Invalid switch - "' & $CmdLine[$iLoop] & "." & @CRLF)
-				Exit 1
+				If @Compiled Then ; support for running not compiled scirpt - mLipok
```
Comment (bl-ue) [inline WhyNotWin11.au3:112]: I'd word it thus:
```suggestion
				If @Compiled Then ; support for running non-compiled script - mLipok
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

File: WhyNotWin11.au3 (hunk 1 above)
Comment (micwoj92) [reply WhyNotWin11.au3]: As this was no code change, added.

**Comment by rcmaehl** [comment]
Not acting like an issue on AutoIt 3.3.14.5 but adding in case there's issues with older versions.

**Review by rcmaehl — APPROVED** [review]

**Comment by micwoj92** [comment]
> Not acting like an issue on AutoIt 3.3.14.5 but adding in case there's issues with older versions.

I don't think we should support it then. Only latest version + latest beta. If someone needs to use au3 version that old, then probably he can't run win11 anyway.

**Pull request merged**